miniz/miniz.c

```diff
@@ -187,6 +187,8 @@ const char *mz_version(void)
 
 #ifndef MINIZ_NO_ZLIB_APIS
 
+#ifndef MINIZ_NO_DEFLATE_APIS
```

Comment:
Turns out we need DEFLATE for PNG saving and INFLATE for ZIP file support, and we need the ZLIB API for extended nodes support, so there is not much more to trim down from this library.